collection/sreevani.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "sreevani",
+    "meaning": "Goddess Saraswati",
+    
+    "translations": {
+        "sree": "スリー",
```

Comment:
Can you please use the correct ISO code for translations?
The list can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).